star-dash/star-dash/Rendering/MTKRenderer/JoystickView.swift

```diff
@@ -0,0 +1,74 @@
+import UIKit
+
+/**
+ `JoystickView` is represents the joystick which the user drags with
+ to move the character.
+ */
+class JoystickView: UIView {
+
+    let joystickBackground: UIView
+    let joystickControl: UIView
+
+    let joystickSize: CGFloat = 100
```

Comment:
probably should include the size of the joystick in init as we may want to change the joy stick size from single to local multiplayer